tests/xx-epub-template/OPS/package.opf

```diff
@@ -1,11 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
   <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
-    <dc:title>Title</dc:title>
-    <dc:creator>Creator</dc:creator>
-    <dc:date>2021-01-01</dc:date>
-    <dc:identifier id="pub-id">NOID</dc:identifier>
+    <dc:identifier id="pub-id">TODO: spec-anchor</dc:identifier>
+    <dc:title>TODO: Concise Test Title</dc:title>
+    <dc:description>
+        TODO: A longer description of the test for the generated test report.
+    </dc:description>
+    <dc:coverage>TODO: Report category</dc:coverage>
+    <meta property="dcterms:isReferencedBy">TODO: https://www.w3.org/TR/epub-33/#spec-anchor</meta>
+    <meta property="dcterms:isReferencedBy">TODO: https://www.w3.org/TR/epub-rs-33/#spec-anchor</meta>
+
+    <dc:creator>W3C EPUB 3 Working Group</dc:creator>
     <dc:language>en</dc:language>
+    <dc:date>2021-01-01</dc:date>
     <meta property="dcterms:modified">2021-01-01T00:00:00Z</meta>
```

Comment:
I have this organized in two sections — stuff up top that needs to be modified for every test, in the order documented at https://github.com/w3c/epub-tests/tree/instructions#metadata, followed by these properties at the bottom that (generally) don't need to be changed.

How do we feel about a) that and b) the general order? Open to changing it.

Reply:
This looks o.k. to me. My script is not order dependent, b.t.w., so whatever looks o.k. works.

Reply:
I decided to go with strict alphabetization. It's the only scheme that isn't subjective and anyone can understand (even if it is not _optimal_ to some eyes).